src/fusion/fusion.rs

```diff
@@ -190,7 +190,7 @@ impl FusionPair {
 #[pymethods]
 impl Fusion {
     #[new]
-    fn new(state: State) -> Self {
+    pub fn new(state: State) -> Self {
```

Comment:
Is this okay? Not sure if this isn't pub for something to do with the pymethods.